src/library_pthread.js

```diff
@@ -479,22 +476,30 @@ var LibraryPThread = {
 
     getNewWorker: function() {
       if (PThread.unusedWorkers.length == 0) {
-#if !PROXY_TO_PTHREAD && PTHREAD_POOL_SIZE_STRICT
+// PTHREAD_POOL_SIZE_STRICT should show a warning and, if set to level `2`, return from the function.
+#if PTHREAD_POOL_SIZE_STRICT && ASSERTIONS || PTHREAD_POOL_SIZE_STRICT == 2
+// However, if we're in Node.js, then we can create new workers on the fly and PTHREAD_POOL_SIZE_STRICT
+// should be ignored altogether.
```

Comment:
Where do we actually create new workers on the fly in Node? Is it the normal `new Worker` but that happens to work synchronously in Node?

Reply:
Yep.